docs/cli.md

```diff
@@ -152,710 +163,464 @@ keytabFile=file.keytab
-```
+`gcli catalog update --name <old-name> --rename <new-name>`
+:Rename a catalog.
 
-#### Display a catalog's properties
+`gcli catalog update --name <name> --comment "<new comment>"`
+: Change the comment for a catalog.
 
-```bash
-gcli catalog properties --name catalog_mysql
-```
-
-#### Set a catalog's property
+`gcli catalog properties --name <name>`
+: List the properties for a catalog.
 
-```bash
-gcli catalog set --name catalog_mysql --property test --value value
-```
+`gcli catalog set --name <name> --property <property> --value <value>`
+: Set a property for a catalog.
 
-#### Remove a catalog's property
-
-```bash
-gcli catalog remove --name catalog_mysql --property test
-```
+`gcli catalog remove --name <name> --property <property>`
+: Remove a property from a catalog.
 
 ### Schema commands
 
-#### Show all schemas in a catalog
-
-```bash
-gcli schema list --name catalog_postgres
-```
-
-#### Show schema details
-
-```bash
-gcli schema details --name catalog_postgres.hr
-```
-
-#### Show schema audit information
-
-```bash
-gcli schema details --name catalog_postgres.hr --audit
-```
+`gcli schema create --name <schema>`
+: Create a schema.
 
-#### Create a schema
+`gcli schema properties --name <schema> -i`
+: Display the properties about a schema.
 
-```bash
-gcli schema create --name catalog_postgres.new_db
-```
+`gcli schema list --name <catalog>`
+: List all schemas in the specified catalog.
 
-#### Display schema properties
+`gcli schema details --name <schema>`
+: Show the details about schemas
 
-```bash
-gcli schema properties --name catalog_postgres.hr -i
-```
+`gcli schema details --name <schema> --audit`
+: Show schema audit information
 
-Setting and removing schema properties is not currently supported by the Java API or the Gravitino CLI.
+:::note
+Setting and removing schema properties is currently not supported in the Java API
+or the Gravitino CLI.
+:::
 
 ### Table commands
 
-When creating a table the columns are specified in CSV file specifying the name of the column, the datatype, a comment, true or false if the column is nullable, true or false if the column is auto incremented, a default value and a default type. Not all of the columns need to be specifed just the name and datatype columns. If not specified comment default to null, nullability to true and auto increment to false. If only the default value is specified it defaults to the same data type as the column.
+When creating a table, the columns are specified in a CSV file containing the properties
+of each column, including
+
+- the column name (required)
+- the data type (required)
+- the comment string which defaults to null
+- a string format boolean indicating whether the column can be null (default=true)
+- a string format boolean indicating whether the column is auto-incremented (default=false)
+- the default value
+- the default data type, defaults to the column data type if a default value is specified.
```

Comment:
I think we'd better mention that, because we are not saying
"if a default type is specified".
The defaulting of "type" depends on the setting of "value".
We need to highlight this.

Reply:
You need to specify the column data type and optionally a default value. If you specify the value you can optionally set the data type of that value. If you do not specify the value's default type, it will use the column's data type. They are likely to be the same in 99% of cases, I'd guess.

Reply:
Let me try rephrase this ... you mean the following is true no matter a default value is specified or not, right?

> the default data type, defaults to the column data type

Reply:
No as if you don't specify a default value, there is no default value to have a default type.  If you do specify a default value but not a default type it will default to the column's data type, OR you can specify a default value and a default type.

Reply:
Thanks for the clarification. Then I believe the if-clause is required.

Reply:
The above text "the default data type, defaults to the column data type if a default value is specified." is incorrect because if you specify a default value and data type, it will not default to the column data type.

Reply:
Also (very minor), if you end in a full stop, you probably want to start with a capital and do the same for the other points.

Reply:
Some people would take that sentence as: If you specify a default value, then the default column type defaults to the column data type, whether you specified that or not.

Reply:
> If you specify a default value, then the default column type defaults to the column data type

Yes. That is exactly what I captured from your clarification.
Maybe you can provide a clearer description?

BTW, this logic is ugly and needs some rethinking. Are we sure we want to do this?
You can see I have had a difficult time to understand this logic.
The same difficulty will be a pain for other users/developers.
If we have a logic that is so difficult to explain, I'd suggest we drop this smart hack.

Reply:
I'll come up with some better text. In 99% of cases, the column will be omitted, I think most people will just follow the example and not even notice.